layers.py

```diff
@@ -779,7 +817,10 @@ def forward(
             nei_inds,
             dense_xyz_norm=None,
             sparse_xyz=None,
-            sparse_xyz_norm=None):
+            sparse_xyz_norm=None,
+            inv_neighbors=None,
+            inv_k=None,
+            inv_idx=None):
```

Comment:
Comment of the function should include these 4 variables along with their sizes as well.